src/System.Security.Cryptography.Primitives/ref/System.Security.Cryptography.Primitives.csproj

```diff
@@ -10,11 +10,12 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Release|AnyCPU'" />
   <ItemGroup>
     <Compile Include="System.Security.Cryptography.Primitives.cs" />
+    <Compile Include="System.Security.Cryptography.Primitives.netcoreapp.cs" Condition="'$(TargetGroup)' == 'netcoreapp'" />
```

Comment:
Why do we need it specialized like this?

Reply:
Because the last library I made API changes to wasn't part of netstandard, apparently.  Merged it into the other file.